<launch>
    <param name="file_spotserver" value="$(file pr2_guide)/spotserver.pickle" />

    <node pkg="pr2_guide" type="go-to-spot.l" name="/pr2_guide/go_to_spot" />

    <node pkg="pr2_guide" type="pr2-guide.py" name="/pr2_guide/pr2_guide" />

    <node pkg="pr2_guide" type="spotserver.py" name="/pr2_guide/spotserver"
        args="$(arg file_spotserver)" />
</launch>
